### ISSUE TITLE
e_brush() selects too many points when group_by()

### ISSUE THREAD

**Issue by matthias-a** [JohnCoene/echarts4r#137]
Hi there,

not sure if I use the brush functionality correctly, but for example in 

```
iris %>%
  group_by(Species) %>%
  e_charts(Petal.Length) %>%
  e_scatter(Petal.Width) %>%
  e_brush()
```

if selecting some points, e.g. by box select, other points outside of the selected area get highlighted as well. Refer to screenshot:
![image](https://user-images.githubusercontent.com/26443836/76644286-ff1e7380-6556-11ea-8e84-3b6cc9dce161.png)

Happens to me with CRAN and dev version.

Do you have any idea what could cause this or how I could adapt my code to select only the data points inside the selection area?

Thanks in advance, best regards
Matthias

**Comment by JohnCoene** [comment]
Interesting, I've looked into but I'm afraid I can't tell you why it does this, sorry. It seems to have to do with the underlying JavaScript library.

**Comment by matthias-a** [comment]
Hi and thanks for your response. I tried a little more and read the documentation (...what I probably should have done in the first place, sorry) and the following fits my purpose:

```
iris %>%
  group_by(Species) %>%
  e_charts(Petal.Length) %>%
  e_scatter(Petal.Width) %>%
  e_brush(brush_link = "none")
```

I did not try too much further but according to [brushLink docu](https://echarts.apache.org/en/option.html#brush.brushLink) and a small test with e.g. `brush_link = c(0, 2)` it seems like one might also be able to specify series idices that should be connected. 

So, looks like it was already working fine and this issue can be closed. Sorry once more!
